quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/widget/rest/WidgetController.java

```diff
@@ -0,0 +1,65 @@
+package com.github.reflectoring.infiniboard.quartermaster.widget.rest;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.github.reflectoring.infiniboard.packrat.source.SourceData;
+import com.github.reflectoring.infiniboard.packrat.widget.WidgetConfig;
+import com.github.reflectoring.infiniboard.quartermaster.widget.domain.WidgetConfigService;
+
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.OK;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+
+@RestController
+@RequestMapping("/api/dashboards/{dashboardId}/widgets")
+public class WidgetController {
+
+    private WidgetConfigService widgetService;
+
+    @Autowired
+    public WidgetController(WidgetConfigService widgetService) {
+        this.widgetService = widgetService;
+    }
+
+    @RequestMapping(value = "/{widgetId}", method = GET)
+    public ResponseEntity<WidgetConfigResource> getWidget(@PathVariable Integer dashboardId,
+                                                          @PathVariable String widgetId) {
+        WidgetConfig widgetConfig = widgetService.loadWidget(widgetId);
+
+        if (widgetConfig == null) {
+            return new ResponseEntity<>(NOT_FOUND);
+        }
+
+        WidgetConfigResourceAssembler assembler = new WidgetConfigResourceAssembler(dashboardId);
+        WidgetConfigResource          resource  = assembler.toResource(widgetConfig);
+        return new ResponseEntity<>(resource, OK);
+    }
+
+    @RequestMapping(method = POST)
+    public ResponseEntity<WidgetConfigResource> createWidget(@PathVariable Integer dashboardId,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)